libs/components/editor-core/src/editor/editor.ts

```diff
@@ -479,7 +479,22 @@ export class Editor implements Virgo {
             return false;
         });
     }
-
+    public copyBlock = async (blockId: string, parentId: string) => {
```

Comment:
Is the parent id is optional?

Reply:
This doesn't look like a copy of the block, but rather a new blcock created and inserted into the parent block